exporter/ibft/reader.go

```diff
@@ -68,35 +67,41 @@ func (r *reader) Sync() error {
 		time.Sleep(1 * time.Second)
 	}()
 	netWaitGroup.Wait()
-	hs := ibftsync.NewHistorySync(r.logger, r.validatorShare.PublicKey.Serialize(), nil, r.network, r.storage, r.validateDecidedMsg) // TODO need to pass identifier
-	return hs.Start()
+	// creating HistorySync and starts it
+	identifier := []byte(validator.IdentifierFormat(r.validatorShare.PublicKey.Serialize(), beacon.RoleTypeAttester))
+	hs := ibftsync.NewHistorySync(r.logger, r.validatorShare.PublicKey.Serialize(), identifier,
+		r.network, r.storage, r.validateDecidedMsg)
+	err = hs.Start()
+	if err != nil {
+		r.logger.Error("could not sync validator's data", zap.Error(err))
+	}
+	return err
 }
 
 // Start starts the network listeners
 func (r *reader) Start() {
-	r.listenToNetworkDecidedMessages()
+	//r.listenToNetworkDecidedMessages()
 }
 
-// listenToNetworkDecidedMessages listens for decided messages
```

Comment:
@amir-blox should this be commented out?

Reply:
It was removed due to irrelevancy